pipelines/validation/h1c_idr3_2/h1c_idr3_v2_validation.toml

```diff
@@ -78,7 +89,19 @@ actions = ["EXTRACT_AUTOS",
            # "CLEAN_FILES", # TODO: update this and put it back in
           ]
 
+[UNCALIBRATE]
+args = ["{basename}",
+        "${UNCALIBRATE_OPTS:sky_cmp}",
+        "${UNCALIBRATE_OPTS:path_to_sim_configs}",
+        "${UNCALIBRATE_OPTS:path_to_sim_files}",
+        "${UNCALIBRATE_OPTS:path_to_scripts}",
+        "${UNCALIBRATE_OPTS:path_to_data_files}",
+        "${UNCALIBRATE_OPTS:save_dir_base}",
+       ]
+
 [EXTRACT_AUTOS]
+prereqs = "UNCALIBRATE"
+prereq_chunk_size = "all"  # Does this need to be set?
```

Comment:
As long as different uncalibrate jobs are completely independent, this isn't necessary.

Reply:
Cool, thanks for the clarification!